src/components/fair/booth.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+
+import Artworks from "../artwork_grid"
+import Button from "../buttons/ghost"
+import FollowButton from "../follow"
+import Text from "../text"
+
+import * as fonts from "../../assets/fonts"
+
+const PageSize = 8
+
+interface Props extends RelayProps, React.HTMLProps<FairBooth> {
+  show: any
```

Comment:
Ah good point, I usually add the normal props first and then end up adding `RelayProps` in later, so this is just a result of forgetting to remove this one. Pretty sure it gets overridden though, to answer your question.